api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -1253,6 +1253,24 @@ public void updateOrderFulfillerStatus_shouldEditFulfillerStatusInOrder() {
 		assertEquals(updatedOrder.getFulfillerComment(), commentText);
 	}
 	
+	/**
+	 * @see OrderService#updateOrderFulfillerStatus(org.openmrs.Order,
+	 *      Order.FulfillerStatus, String, String)
+	 */
+	@Test
+	public void updateOrderFulfillerStatus_shouldEditFulfillerStatusWithAccessioNumberInOrder() {
+		Order originalOrder = orderService.getOrder(111);
+		String commentText = "We got the new order";
+		String accessionNumber = "12345";
+		assertNotEquals(originalOrder.getAccessionNumber(), 0);
```

Comment:
Minor point, but shouldn't we really be testing here that the accession number is not "12345"?

Reply:
@mogoodrich sure..i have fixed it